### PR TITLE
Add analyze output to bug-report

### PR DESCRIPTION
**Pull request by ostromart**

### CODE REVIEW

**Review by stewartbutler — APPROVED** [review]
LGTM

File: tools/bug-report/pkg/content/content.go [hunk 1]
```diff
@@ -180,6 +195,46 @@ func GetNetstat(p *Params) (map[string]string, error) {
 	return retMap("netstat", out, err)
 }
 
+// GetAnalyze returns the output of istioctl analyze.
+func GetAnalyze(p *Params) (map[string]string, error) {
+	out := make(map[string]string)
+	sa := local.NewSourceAnalyzer(schema.MustGet(), analyzers.AllCombined(),
+		resource.Namespace(p.Namespace), resource.Namespace(p.IstioNamespace), nil, true, 5*time.Minute)
```
Comment (howardjohn) [inline tools/bug-report/pkg/content/content.go:202]: is this analyzing all namespaces?

Comment (ostromart) [reply tools/bug-report/pkg/content/content.go:202]: yes, except kube-system and kube-public

**Review by ericvn — APPROVED** [review]

**Pull request merged**